### ISSUE TITLE
Usability of "Communication problem: Take note of any unsaved data, and click here or press ESC to continue"

### ISSUE THREAD

**Issue by jindrichmynarz** [UnifiedViews/Core#481]
The warning *"Communication problem: Take note of any unsaved data, and click here or press ESC to continue"* could be improved.

Asking the user to *"take note of any unsaved data"* is meaningless because, at this point, it is not possible to take note of any unsaved data, since configurations of DPU instances cannot be opened. Either remove this message or make it possible to *"take note of any unsaved data"* in some way.

Both clicking on the warning and pressing ESC have the same result: pipeline is reloaded. If the warning indicates there are 2 actions possible, they should be distinct. This may be related to #370.

Context: ODN 1.1.0

**Comment by eea03** [comment]
These are default Vaadin texts.
Localization of these messages was implemented in https://github.com/UnifiedViews/Core/pull/460

English texts were just copied but it is easy to customize them now.
Clicking and ESC behavior is also Vaadin core functionality so I would prefer adjusting the message to be clear that both actions have the same result

**Comment by tomas-knap** [comment]
Yes, the simplest we can do is to customize the messages. So to solve this issue, please adjust the text so that it is not confusing and directly says that your data will be lost.
